### PR TITLE
Fix access to RACS data

### LINKED ISSUE #357: Cannot find RACS images or noise maps
```
example_query = Query(
    coords=SkyCoord(45.733478, -50.825550, unit=u.deg),
    crossmatch_radius=5.,
    epochs='all'
)
example_query.find_sources()
```

Throws the following warnings

```
/data/vast-survey/VAST/release/EPOCH00/COMBINED/STOKESI_IMAGES/RACS_0304-50A.EPOCH00.I.conv.fits does not exist! Unable to create postagestamps
/data/vast-survey/VAST/release/EPOCH00/COMBINED/STOKESI_RMSMAPS/noiseMap.RACS_0304-50A.EPOCH00.I.conv.fits does not exist! Unable to get noise maps.
```

### LINKED ISSUE #359: Query initialisation: RACS warning is thrown twice.
Any query involving both RACS epochs has the RACS warning thrown twice, i.e.

```
RACS data selected!
Remember RACS data supplied by VAST is not final and results may vary.
RACS data selected!
Remember RACS data supplied by VAST is not final and results may vary.
```

### PR DESCRIPTION
**Pull request by ddobie**
Fix #357.
Fix #359.

### CODE REVIEW

**Comment by ddobie** [comment]
For what it's worth, this error is missed by the testing because we only test the naming for a single epoch. That should probably be fixed in the future.

File: vasttools/query.py [hunk 1]
```diff
@@ -1836,14 +1835,15 @@ def _add_files(self, row: pd.Series) -> Tuple[str, str, str]:
 
         img_dir = "STOKES{}_IMAGES".format(self.settings['stokes'])
         rms_dir = "STOKES{}_RMSMAPS".format(self.settings['stokes'])
+        field = row.field.replace('RACS', 'VAST')
```
Comment (ajstewart) [inline vasttools/query.py:1838]: I'm wondering, if this is the solution, why is the `RACS` there in the first place? Can it be fixed earlier?

**Comment by ajstewart** [comment]
> For what it's worth, this error is missed by the testing because we only test the naming for a single epoch. That should probably be fixed in the future.

I guess this probably hinges on my point above on whether RACS is needed to be there at this point or not.

File: vasttools/query.py (hunk 1 above)
Comment (ddobie) [reply vasttools/query.py:1838]: We can rename the fields when they're loaded, or even in the csv files themselves.

However, I don't think that's optimal. The field was observed as RACS_XXXX-XXXX, not VAST_XXXX-XXXX, and I think the field name should reflect that. It's minor, but I think it's important.

Comment (ajstewart) [reply vasttools/query.py:1838]: Ok, so they are only sorted in the data system with the VAST tag?

Comment (ddobie) [reply vasttools/query.py:1838]: Yes - the file names all use a VAST prefix regardless of whether the data is from VAST or RACS. I'm not sure I agree with that, but it is what it is. I think the biggest thing is making sure people know what to cite and acknowledge (i.e. they should be citing RACS if they're using RACS data).

**Review by ajstewart — DISMISSED** [review]

**Pull request merged**